Dockerfile

```diff
@@ -21,7 +10,24 @@ COPY routing.py routing.py
 COPY generated generated
 COPY main.py .
 COPY requirements.txt .
-RUN pip3 install --no-cache-dir -r requirements.txt
 
-# configure default command for the image
+# install build-essential needed by `pip3 install` on ARM64,
```

Comment:
I think that this RUN command should be above "install SLA dashboard app" section so that the developer can quickly rebuild the image with modified code without triggering apt-get commands. Only "requirements.txt" is needed for pip install